### PR TITLE
docs: update changelog for v0.7.0

### PR DESCRIPTION
**Pull request by aschmahmann**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,529 @@
...
+  - Integration test update for go versions.
+- github.com/whyrusleeping/cbor-gen (v0.0.0-20200123233031-1cdf64d27158 -> v0.0.0-20200402171437-3d27c146c105):
+  - Handle Nil values for cbg.Deferred ([whyrusleeping/cbor-gen#14](https://github.com/whyrusleeping/cbor-gen/pull/14))
+  - add name of struct field to error messages
+  - Support uint64 pointers ([whyrusleeping/cbor-gen#13](https://github.com/whyrusleeping/cbor-gen/pull/13))
+  - int64 support in map encoders ([whyrusleeping/cbor-gen#12](https://github.com/whyrusleeping/cbor-gen/pull/12))
+  - Fix uint64 typed array gen ([whyrusleeping/cbor-gen#10](https://github.com/whyrusleeping/cbor-gen/pull/10))
+  - Fix cbg self referencing import path ([whyrusleeping/cbor-gen#8](https://github.com/whyrusleeping/cbor-gen/pull/8))
+
+### Contributors
```
Comment (aschmahmann) [inline CHANGELOG.md:469]: Note for posterity: There were a couple modifications made to the mkreleaselog script for this run through:
1) Some code generated files from go-libp2p-asn-util were manually ignored as they didn't follow a nice pattern for detecting codegen'd files.
2) Commits to go-multiaddr were ignored do to the overwhelming number of commits that were ported from `go-multiaddr-net`. Nobody's numbers should have been affected in a meaningful way.

**Comment by aschmahmann** [comment]
@Stebalien do you generally clean up the changelog more than this such as removing commits that are not of interest to the community (e.g. updating webui multiple times, tiny bug fixes to features introduced during the release, etc.)?

**Comment by Stebalien** [comment]
I generally leave all changes except dep updates.

> Also, WDYT about merging this after we merge the release branch back into master instead of before? This means we don't have to deal with the changelog of for a release until the release is actually finalized.

I'm not sure I understand. You mean don't merge this into the release branch? This change should go in the released source tarball (and should be inside the tagged release).

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,544 @@
...
+```bash
+ipfs dag stat bafybeihpetclqvwb4qnmumvcn7nh4pxrtugrlpw4jgjpqicdxsv7opdm6e # the IPFS webui
+Size: 30362191, NumBlocks: 346
+```
+
+#### Plugin build changes
+
+We have changed the build flags used by the official binary distributions on dist.ipfs.io (or `/ipns/dist.ipfs.io`) to use the simpler and more reliable `-trimpath` flag instead of the more complicated and brittle `-asmflags=all=-trimpath="$(GOPATH)" -gcflags=all=-trimpath="$(GOPATH)"` flags, however the build flags used by default in go-ipfs remain the same.
+
+The scripts in https://github.com/ipfs/go-ipfs-example-plugin have been updated to reflect this change. This is a breaking change to how people have been building plugins against the dist.ipfs.io binary of go-ipfs and plugins should update their build processes accordingly see ipfs/go-ipfs-example-plugin#9 for details.
```
Comment (jacobheun) [inline CHANGELOG.md:56]: The markdown doesn't get the PR link automatically, so it needs to be long hand.

```suggestion
The scripts in https://github.com/ipfs/go-ipfs-example-plugin have been updated to reflect this change. This is a breaking change to how people have been building plugins against the dist.ipfs.io binary of go-ipfs and plugins should update their build processes accordingly see https://github.com/ipfs/go-ipfs-example-plugin/pull/9 for details.
```

**Review by jacobheun — APPROVED** [review]
Just one link issue, otherwise lgtm

**Pull request merged**